pages/drop-checker/[chain]/[collection-id].vue

```diff
@@ -0,0 +1,308 @@
+              </td>
+            </tr>
+          </table>
+        </div>
+
+        <div>
+          <p>Data from indexer:</p>
+          <table
+            v-if="collection && collection.meta"
+            class="table-fixed w-full">
+            <tr>
+              <th class="text-k-primary">Key</th>
+              <th class="text-k-primary">Value</th>
+            </tr>
+            <tr
+              v-for="drop in Object.entries(collection)"
+              :key="drop[0]"
+              :class="{
+                hidden: drop[0] === 'meta' || drop[0] === '__typename',
+              }">
+              <td class="align-middle w-40">{{ drop[0] }}</td>
+              <td class="align-middle break-words">
+                <div v-if="drop[0] === 'metadata'">
+                  :
+                  <NuxtLink
+                    class="border-b"
+                    :to="sanitizeIpfsUrl(drop[1]?.toString())"
+                    target="_blank">
+                    🔗 {{ sanitizeIpfsUrl(drop[1]?.toString()) }}
+                  </NuxtLink>
+                </div>
+                <div v-else>: {{ drop[1] }}</div>
+              </td>
+            </tr>
+            <tr
+              v-for="meta in Object.entries(collection.meta)"
+              :key="meta[0]"
+              :class="{ hidden: meta[0] === '__typename' }">
+              <td class="align-middle w-40">meta.{{ meta[0] }}</td>
+              <td class="align-middle break-words">
+                <div v-if="meta[0] === 'image'">
+                  :
+                  <img
+                    :src="sanitizeIpfsUrl(meta[1]?.toString())"
+                    width="100" />
+                </div>
+                <div v-else-if="meta[0] === 'id'">
+                  :
+                  <NuxtLink
+                    :to="sanitizeIpfsUrl(meta[1]?.toString())"
+                    target="_blank"
+                    class="border-b">
+                    🔗 {{ sanitizeIpfsUrl(meta[1]?.toString()) }}
+                  </NuxtLink>
+                </div>
+                <div v-else>: {{ meta[1] }}</div>
+              </td>
+            </tr>
+          </table>
+        </div>
+      </div>
+    </neo-collapse>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { NeoButton, NeoCollapse, NeoIcon } from '@kodadot1/brick'
+import { keccakAsHex } from '@polkadot/util-crypto'
+import collectionById from '@/queries/subsquid/general/collectionById.query'
+import { getDrops } from '@/services/fxart'
+import { formatAmountWithRound } from '@/utils/format/balance'
+import type { Prefix } from '@kodadot1/static'
+
+const { params } = useRoute()
+const chain = params.chain.toString()
+const collectionId = params['collectionid'].toString()
+
+const { chainSymbol, decimals } = useChain()
+
+// get data from database
+const data = await getDrops({
+  chain: [chain],
+  limit: 10,
+})
+const drops = data.filter((drop) => drop.collection === collectionId)[0]
```

Comment:
It only gets information from the first 10 drops. Could we make it 100 or so?

Reply:
oh, correct, let's use 100 for now
later on, we need to fetch drop by chain & collection id: https://github.com/kodadot/private-workers/issues/200